_data/toc/php-developer-guide.yml

```diff
@@ -258,11 +258,14 @@ pages:
           url: /extension-dev-guide/configuration/importers.html
 
     - label: Framework
-      include_versions: ["2.2", "2.3"]
       children:
         - label: Serialize Library
+          include_versions: ["2.2", "2.3"]
```

Comment:
Please revert this change. The Framework section isn't supposed to exist in 2.1. Moving this parameter causes the site to build an empty framework section in 2.1.